opentelemetry-sdk/tests/trace/export/test_export.py

```diff
@@ -439,6 +441,7 @@ def test_batch_span_processor_scheduled_delay(self):
 
         span_processor.shutdown()
 
+    @flaky(max_runs=3, min_passes=1)
```

Comment:
I have see this show flakiness only on pypy. Is it possible to enable this for only that env?

Reply:
Yes, fixed :v: